.circleci/config.yml

```diff
@@ -41,6 +41,8 @@ jobs:
       - run: make ci-lint
       - run: make test | go-junit-report > $TEST_RESULTS/gotest/report.xml
       - run: make integration-test
+      - setup_remote_docker
```

Comment:
Generally you probably want to do this as the first step in the build? I'd move it higher, as a result. Mostly because it's environment config?

Reply:
Done in 4a9c74d